tests/scripts/thread-cert/v1_2_test_csl_transmission.py

```diff
@@ -71,6 +72,16 @@ def test(self):
         self.assertTrue(self.nodes[LEADER].ping(self.nodes[SSED_1].get_rloc()))
         self.simulator.go(5)
 
+        self.nodes[SSED_1].set_csl_channel(SECOND_CHANNEL)
+        self.nodes[LEADER].set_csl_channel(SECOND_CHANNEL)
```

Comment:
is it necessary to configure the CSL channel for leader?

Reply:
A mistake here. It should be `set_channel`. The purpose is to test, without specifying csl channel, csl tx works properly when both devices change their channels. Changing channels can only be done when the devices are stopped. I added a logic to stop, set channel and restart.

Reply:
`stop, set_channel and then start` doesn't make any difference with the procedure before stop as by default both devices work on the same default channel 11.

To test the csl channel (when unspecified) changes along with pan channel, the proper way is to change the channel of running stack via pending dataset.